concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/00 A Drop-Down Editor Does Not Show Data.md

```diff
@@ -1,9 +1,9 @@
-This section describes the issues related to the following drop-down editors:
+This section describes the solutions to the issues (an empty edit box or an empty dropdown list) related to the following drop-down editors:
```

Comment:
I tried to incorporate issue descriptions instead of the word "issues". Not that parentheses are bad, but this is a pretty long and complex sentence. 

```suggestion
This section describes what you can do if the following drop-down editors display an empty edit box or an empty dropdown list:
```